src/components/Settings/SettingsSortGroups.vue

```diff
@@ -0,0 +1,77 @@
+<!--
+  - @copyright Copyright (c) 2020 Uriel Fontan <urielfontan2002@gmail.com>
+  -
+  - @author Uriel Fontan <urielfontan2002@gmail.com>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<template>
+	<div class="sort-groups">
+		<Multiselect
+			id="sort-by"
+			:value="orderGroupsOption"
+			:searchable="false"
+			:allow-empty="false"
+			:options="optionsGroups"
+			:custom-label="formatSortByLabel"
+			track-by="key"
+			label="label"
+			@input="sortGroups" />
+	</div>
+</template>
+
+<script>
+
+export default {
+	name: 'SettingsSortGroups',
+
+	computed: {
+		/* Order Keys */
+		optionsGroups() {
+			return [
+				{
+					label: t('contacts', 'Amount'),
+					key: 'amount',
+				},
+				{
+					label: t('contacts', 'Alphabetical Order'),
+					key: 'alphabetically',
+				},
+			]
+		},
+		/* Current order Key */
+		orderGroups() {
+			return this.$store.getters.getOrderGroups
+		},
+		orderGroupsOption() {
+			return this.optionsGroups.find(option => option === this.orderGroups)
```

Comment:
Let's fallback to default

```suggestion
			return this.optionsGroups.find(option => option === this.orderGroups) || this.optionsGroups[0]
```

Reply:
mmm weird because this.orderGroups should return the state.orderGroups, maybe its null?